src/region.js

```diff
@@ -7,6 +7,8 @@
 // http://lostechies.com/derickbailey/2011/12/12/composite-js-apps-regions-and-region-managers/
 
 Marionette.Region = Marionette.Object.extend({
+  _cid: 'mnr',
```

Comment:
hmm "private" or public..

I suppose there's some arguments that others may want to change the `cid` for their extensions of things.. but at the same time this is not how view cids or model cids work.  if it is public would view / collectionview then follow suit with `mnv_` `mncv_`?